tests/foreman/cli/test_puppetmodule.py

```diff
@@ -0,0 +1,115 @@
+"""Test class for PuppetModule CLI
+
```

Comment:
I think something in my rebase caused me try bring this file back. Should I delete this?